docs/HowTo/Configure/Consensus-Protocols/IBFT.md

```diff
@@ -10,7 +10,8 @@ IBFT 2.0.
 !!! warning
 
     Configure your network to ensure you never lose 1/3 or more of your validators. If more
-    than 1/3 of validators stop participating, new blocks are no longer be created.
+    than 1/3 of validators stop participating, new blocks are no longer created and
+    serious issues can occur.
```

Comment:
Could it be more specific about what kind of serious issues? How serious, what can be broken?
Otherwise, perhaps "unexpected issues" would be better?

Reply:
@bgravenorst - to Nico's point this could be more explicit: 

"your network will stall and may take significant time to recover once nodes are restarted."

Reply:
Done